vignettes/ParametricCopulaFamilies.Rmd

```diff
@@ -0,0 +1,163 @@
+---
+title: "Parametric Copula Families and Their Functions"
+params:
```

Comment:
Don't need the `params` field since this is just for writing parametrized reports.